src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -569,6 +569,11 @@ public async Task ReplyForbiddenWithWwwAuthenticateHeaderAsync(IEnumerable<strin
 
             string parameterString = string.Join(", ", parameters.Select(p => $"{p.Key}=\"{p.Value}\""));
 
+            if (CurrentHttpContext == null)
```

Comment:
I don't think we want to do this, as for blazor, this will be null.

Reply:
Good point. Suggestions? Without context, this method's kind of pointless. Will need a workaround for Blazor then, right?

Reply:
that's what we've been working on.

Reply:
We now have a work around for Blazor, @pmaytak 
See [Managing incremental consent and conditional access](https://github.com/AzureAD/microsoft-identity-web/wiki/Managing-incremental-consent-and-conditional-access)

Reply:
Created #380 to address this.